### DISCUSSION TITLE
Not getting a good fit in the UV region

### DISCUSSION THREAD

**Discussion by rnbhutkar22** [mboquien/cigale#111] [Q&A]
Hello,

I’m encountering an issue where I’m not getting a good fit in the UV part. After adjusting some parameters, I was able to get a good fit in the UV, but it caused issues in the optical and IR regions. I'm not getting a good fit in the X-ray data either (though X-ray data is optional in my analysis). I’m unable to pinpoint the source of the problem. Any help would be greatly appreciated.
I’ve attached both plots for your reference.

Thank you!

[3C303.1_best_model_1.pdf](https://github.com/user-attachments/files/18158783/3C303.1_best_model.pdf)
[3C303.1_best_model_2.pdf](https://github.com/user-attachments/files/18158788/3C303.1_best_model.pdf)

Input files can be found here:
[https://drive.google.com/drive/folders/1oXOdesUnoH_Bh0n-JhUGSszBd0fSRCCb?usp=sharing](https://drive.google.com/drive/folders/1oXOdesUnoH_Bh0n-JhUGSszBd0fSRCCb?usp=sharing)

**Comment by Guang91** [comment]
It looks like your AGN parameters are quite limited.
Have you tried exploring more parameters like EBV, delta_AGN, frac_AGN, oa,
i?

Cheers,
Guang
